src/earn/EarnHome.test.tsx

```diff
@@ -0,0 +1,171 @@
+      networkId: NetworkId['arbitrum-one'],
+      tokenId: pools[0].tokens[0],
+      address: mockArbArbAddress,
+      symbol: 'ARB',
+      decimals: 18,
+      imageUrl:
+        'https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/ARB.png',
+      balance: '0',
+      priceUsd: '1.2',
+      priceFetchedAt: Date.now(),
+    }
+    const { getByTestId, getAllByTestId } = render(
+      <Provider
+        store={createMockStore({
+          tokens: {
+            tokenBalances: {
+              [pools[0].tokens[0]]: mockPoolToken,
+            },
+          },
+        })}
+      >
+        <MockedNavigator
+          component={EarnHome}
+          params={{
+            activeEarnTab: EarnTabType.OpenPools,
+          }}
+        />
+      </Provider>
+    )
+
+    expect(getByTestId(`PoolCard/${pools[0].poolId}`)).toBeTruthy()
+
+    const tabItems = getAllByTestId('Earn/TabBarItem')
+    expect(tabItems).toHaveLength(2)
+    expect(tabItems[0]).toHaveTextContent('openPools')
+    expect(tabItems[1]).toHaveTextContent('myPools')
+  })
+
+  it('correctly shows pool under my pools if has balance', () => {
+    const mockPoolToken = {
+      name: 'Arbitrum',
+      networkId: NetworkId['arbitrum-one'],
+      tokenId: pools[0].poolTokenId,
+      address: mockArbArbAddress,
+      symbol: 'ARB',
+      decimals: 18,
+      imageUrl:
+        'https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/ARB.png',
+      balance: '10',
+      priceUsd: '1.2',
+      priceFetchedAt: Date.now(),
+    }
+    const { getByTestId, queryByTestId, getByText } = render(
+      <Provider
+        store={createMockStore({
+          tokens: {
+            tokenBalances: {
+              [pools[0].poolTokenId]: mockPoolToken,
+              [`${NetworkId['arbitrum-one']}:0xaf88d065e77c8cc2239327c5edb3a432268e5831`]: {
+                name: 'USDC',
+                networkId: NetworkId['arbitrum-one'],
+                tokenId: `${NetworkId['arbitrum-one']}:0xaf88d065e77c8cc2239327c5edb3a432268e5831`,
+                address: '0xaf88d065e77c8cc2239327c5edb3a432268e5831',
+                symbol: 'USDC',
+                decimals: 18,
+                imageUrl:
+                  'https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/USDC.png',
+                balance: '1',
+                priceUsd: '1',
+                priceFetchedAt: Date.now(),
+              },
+            },
+          },
+        })}
+      >
+        <MockedNavigator
+          component={EarnHome}
+          params={{
+            activeEarnTab: EarnTabType.OpenPools,
+          }}
+        />
+      </Provider>
+    )
+
+    expect(queryByTestId(`PoolCard/${pools[0].poolId}`)).toBeFalsy()
+    fireEvent.press(getByText('earnFlow.poolFilters.myPools'))
+    expect(getByTestId(`PoolCard/${pools[0].poolId}`)).toBeTruthy()
+  })
+
+  it('correctly shows correct networks, tokens under filters', () => {
```

Comment:
This might be difficult with the current hardcoded single pool. There isn't any way to specify for the home screen to show more pools, so the pools being shown wouldn't change at all.

Reply:
you can mock the pools.ts file with jest right?